deployment/README.md

```diff
@@ -1,21 +1,24 @@
 # Distributor GCP Deployment
 
-This terraform script configures a GCP project with a running distributor.
+This directory contains configuration-as-code to deploy the distributor to GCP.
+ - `modules`: a terraform module to deploy the distributor
+ - `live`: terragrunt configurations for deploying to different environments
+
+Deploying this requires installation of:
+ - `terraform`
+ - `terragrunt`
```

Comment:
Can you add a link to where/how to install? `terragrunt` doesn't seem to be a common package in distros.

Reply:
Done.